chapters/en/slides/module4/module4_01.md

```diff
@@ -0,0 +1,90 @@
+---
+type: slides
+title: 'module4_01'
+---
+
+# Viewing the history of a project
+
+---
+# Viewing the history of a project
+
+There are two ways you can view the Git history of a project:
+
+**In the remote**
+1. On GitHub through the repo's code commit view
+**In your computer**
+2. On Jupyter Lab through the repo's code commit view
+3. On your local machine using `git log`
```

Comment:
Since you have "In your computer" on line 15, saying "On your local machine" sounds redundant.